protobufs/livekit_rtc.proto

```diff
@@ -479,6 +538,10 @@ message RequestResponse {
     QUEUED = 4;
     UNSUPPORTED_TYPE= 5;
     UNCLASSIFIED_ERROR = 6;
+    INVALID_HANDLE = 7;
+    INVALID_NAME = 8;
+    INVALID_MIME_TYPE = 9;
```

Comment:
@ladvoc Thinking about the checks. What conditions would trigger these two? `INVALID_NAME` and `INVALID_MIME_TYPE`?
- Is name more like length check?
- What mime types are supported? I am guessing we are not going to do any processing using this unlike media where it is used for things like parsing some bits of packet to figure out if it is a key frame etc.